CuePlayer: Update control bar layout and tooltips

DrShpongle opened the pull request:
![image](https://user-images.githubusercontent.com/25487857/121713601-57874c80-cadd-11eb-86f4-144e47360ec1.png)

<img src="https://media.giphy.com/media/p2sjYFtiXA4Zq/giphy.gif" width="150">

DrShpongle commented:
full screen view:

![Image 2021-06-11 at 9 17 25 PM](https://user-images.githubusercontent.com/1519448/121732323-c45d0f80-cafa-11eb-8348-6e70e1addf74.jpg)


and desktop:

![screencapture-egghead-af-3000-video-test-2021-06-11-21_18_00](https://user-images.githubusercontent.com/1519448/121732347-cf17a480-cafa-11eb-8ec8-2200b7986525.png)

DrShpongle commented on src/pages/video-test.tsx:
```diff
@@ -48,19 +48,20 @@ const EggheadPlayer: React.FC<{videoResource: VideoResource}> = ({
               crossOrigin="anonymous"
               className="font-sans"
               poster={videoResource.poster}
+              // className="cueplayer-react-fullscreen"
```
uncomment it and switch to fullscreen to get styles applied

joelhooks replied:
this can be removed
```suggestion
```

joelhooks commented on src/pages/video-test.tsx:
```diff
@@ -174,6 +177,38 @@ const NotesTabContent: React.FC<{cues: VTTCue[]}> = ({cues}) => {
   )
 }
 
+const PlayerContainer: React.ForwardRefExoticComponent<any> = React.forwardRef<
```
I made the container a component so we could dynamically add classes, similar to `Player` - actually just cut and pasted most of those classes, we might make these different? You get the idea.

DrShpongle commented:
- Removed side bar in full screen mode
- added "divider" to control bar so that fullscreen toggle, options bar and speed rate are shifted to the right side

DrShpongle commented:
Here is sketch of how it might look made by @vojtaholik:
![](https://files.slack.com/files-pri/T030CS0QL-F0254246Q5A/image.png)
